core/src/test/scala/org/genivi/sota/core/ImpactResourceSpec.scala

```diff
@@ -39,11 +45,15 @@ class ImpactResourceSpec
 
     val (pkg, device) = f.futureValue
 
+    val affected = Map(device.id -> Seq(pkg.id))
+
+    val route = new ImpactResource(defaultNamespaceExtractor, fakeExternalResolver(affected)).route
+
     Get("/impact/blacklist") ~> route ~> check {
       status shouldBe StatusCodes.OK
-      val resp = responseAs[Seq[Map[Device.Id, PackageId]]]
+      val resp = responseAs[Map[Device.Id, Seq[PackageId]]]
 
-      resp should contain(Map(device.id → pkg.id))
+      resp should contain(device.id -> Seq(pkg.id))
```

Comment:
Is `shouldEqual` not better here, so we ensure that only affected devices are actually returned?